src/helpers/author-archive-helper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * A helper object for author archives.
+ *
+ * @package Yoast\YoastSEO\Helpers
+ */
+
+namespace Yoast\WP\Free\Helpers;
+
+/**
+ * Class Author_Archive_Helper
+ */
+class Author_Archive_Helper {
+
+	/**
+	 * Gets the array of post types that are shown on an author's archive.
```

Comment:
Not sure this description is accurate because we're actually _setting_ this to `post`. This doesn't get the real post types displayed in the author archive. It just sets it to `post` for our convenience.